packages/file_selector/file_selector_android/android/src/main/java/io/flutter/plugins/file_selector/LifeCycleHelper.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.file_selector;
+
+import android.app.Activity;
+import android.app.Application;
+import android.os.Bundle;
+import androidx.annotation.NonNull;
+import androidx.lifecycle.DefaultLifecycleObserver;
+import androidx.lifecycle.LifecycleOwner;
+
+public class LifeCycleHelper
```

Comment:
A flutter provided Lifecycle helper does not exist yet but if you find yourself needing a class like this in the future checkout https://docs.google.com/document/d/1wXB1zNzYhd2SxCu1_BK3qmNWRhonTB6qdv4erdtBQqo/edit?resourcekey=0-WOBqqOKiO9SARnziBg28pg